test/v2/client/test_client_setup.go

```diff
@@ -124,6 +124,40 @@ func skipInCI(t *testing.T) {
 	}
 }
 
+// ensureFileIsPresent checks if a file exists at the given path. If it does not, it downloads the file from the
+// given URL into the given path.
+func ensureFileIsPresent(
+	config *TestClientConfig,
+	path string,
+	url string) error {
+
+	path, err := config.Path(path)
+	if err != nil {
+		return fmt.Errorf("failed to get path: %w", err)
+	}
+
+	_, err = os.Stat(path)
+	if os.IsNotExist(err) {
+		command := make([]string, 3)
+		command[0] = "wget"
+		command[1] = "https://srs-mainnet.s3.amazonaws.com/kzg/g1.point"
```

Comment:
This should be a parameter

Reply:
oops, lol, I was just using the URL in a debug log. Talk about a confusing error to debug.